multicluster/apis/multicluster/v1alpha1/gatewaynode_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2021 Antrea Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GatewayNodeInfo includes basic information of Gateway Node
+type GatewayNodeInfo struct {
+	// The name of Gateway Node
+	Name string `json:"name,omitempty"`
+	// The Gateway IP of the Gateway Node, it might be assigned by user manually
+	// through annotation.
+	GatewayIP string `json:"gatewayIP,omitempty"`
+}
+
+// +genclient
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+
+// GatewayNode is the Schema for the gatewaynodes API
+type GatewayNode struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// The Gateway IP of the Gateway Node, it might be assigned by user manually
+	// through annotation.
+	GatewayIP string `json:"gatewayIP,omitempty"`
+	// Internal IP of Gateway Node
+	InternalIP string `json:"internalIP,omitempty"`
```

Comment:
Do you think add IP here is simpler than getting the tunnel IP from Node resource (via NodeRoute Controller?)?

Reply:
do you mean the NodeRoute Controller in Antrea agent? I feel it will make the scenarios more complicated. for now, ClusterInfo exchange process is independent with Antrea agent.

Reply:
OK, I think it would be better to include internal IP here considering we still need to watch GatewayNode to learn which one is Gateway. so it would be simpler to read the CR to get IP directly.